cli/command/format.go

```diff
@@ -38,7 +38,8 @@ import (
 const (
 	FORMAT_EXAMPLE = `Examples:
   $ curveadm format -f /path/to/format.yaml           # Format chunkfile pool with specified configure file
-  $ curveadm format --status -f /path/to/format.yaml  # Display formatting status`
+  $ curveadm format --status -f /path/to/format.yaml  # Display formatting status
+  $ curveadm format --stop -f /path/to/format.yaml  # stop formatting`
```

Comment:
maybe we should align the comments of command example.

Reply:
> Maybe we should align the comment of command examples.

resolve